### DISCUSSION TITLE
Why don't you use Flask or Django?

### DISCUSSION THREAD

**Discussion by TechStudent10** [Tech-With-Tim/API#44] [Q&A] [answered]
I saw that you were using "Quart" as a framework. Just wondering why didn't you guys didn't opt for Flask or Django.

**Comment by SylteA** [comment, marked as answer]
Quart is (in my option) a better ASGI version of Flask.

Flask and Django are great frameworks but WSGI frameworks are generally slow, and so are Flask and Django.
Other than speed, I like the control we get with closer to application level frameworks like Quart.

**Reply by TechStudent10** [reply]
Thanks for the not so quick reply!

**Reply by SylteA** [reply]
Np, didn't get a notification so😅

**Reply by TechStudent10** [reply]
haha
